visualization/app/codeCharta/ui/presentationModeButton/presentationModeButton.component.html

```diff
@@ -1,6 +1,8 @@
-<i class="presentation-mode-icon fa fa-television"></i>
+<div title="Presentation mode">
```

Comment:
I am not certain what the presentation mode stands for and I guess most users won't. I do know (after using it) that the colors might slightly change.

It would be great to indicate what the presentation mode stands for while hovering over the actual "presentation" icon and to indicate "Enable presentation mode / Disable presentation mode" for the toggle.